contracts/contracts/coordination/Coordinator.sol

```diff
@@ -385,6 +385,13 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         return keccak256(abi.encode(nodes));
     }
 
+    function expectedTranscriptSize(
+        uint16 dkgSize,
+        uint16 threshold
+    ) public pure returns (uint256) {
+        return 40 + (dkgSize + 1) * BLS12381.G2_POINT_SIZE + threshold * BLS12381.G1_POINT_SIZE;
```

Comment:
Just a nitpick but `40` is the only unnamed component of this calculation.  Perhaps it can be assigned to a constant similar to the BLS point sizes?

Reply:
You're right, but the thing is that I don't understand well enough where that 40 comes from. I mean, it's a fixed overhead that comes from rust serialization but I don't have a full explanation for it. I'll update this when I have more info.